packages/paste-core/components/form/src/FormHelperText.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import {Text, safelySpreadTextProps} from '@twilio-paste/text';
+
+export interface FormHelperTextProps extends React.HTMLAttributes<''> {
+  className?: never;
+  hasError?: boolean;
```

Comment:
For the sake of scaling the component in the future, lets imagine we add a warning help text variant, should this be a `variant` enum with just `error` as the only option for now.

I came across this article this week and it got me thinking about it https://kentcdodds.com/blog/stop-using-isloading-booleans

Reply:
I changed this so it has an error variant, like alerts. Do we need the `hasError` prop anymore?

Reply:
Killed the `hasError` prop. Replaced with error variant.